specify/specify7
# Fix camel case issue for field in export feed

Fixes #4650

### Checklist

      and self-explanatory (or properly documented)

### Testing instructions

- delete any existing export feed 
- save 
- create a new export feed using the visual editor 
- switch to xml editor 

- [ ] verify the collectionId and userId are camel case 
- [ ] verify that you can do an export

@grantfitzsimmons edit, also test:

- [ ] Navigate to a database with an existing export feed (one created before this PR)
- [ ] Verify that the export works in `edge`
- [ ] Update the RSS feed from the User Tools menu
- [ ] Verify that the export is completed successfully in `issue-4650`
